frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -358,6 +360,12 @@ const GitSection: React.FC<GitSectionProps> = ({
     values.git.dir,
   ]);
 
+  React.useEffect(() => {
+    if (formReload && values.git.url) {
+      debouncedHandleGitUrlChange(values.git.url, values.git.ref, values.git.dir);
+    }
+  }, [debouncedHandleGitUrlChange, formReload, values.git.dir, values.git.ref, values.git.url]);
```

Comment:
You only need to run this effect when `formReload` changes. For all the other changes we have another effect that runs the validations.

Reply:
Or maybe you can merge this with the other effect. `(!dirty || gitDirTouched || formReload)`.

Reply:
thank you, merged it